swan-lake/development-tutorials/source-code-dependencies/style-guide/annotations-documentation-and-comments.md

```diff
@@ -144,6 +144,53 @@ intro: The sections below include the coding conventions with respect to annotat
     }
     ```
 
+* For record type declarations add the description above the type declaration.
```

Comment:
Shall we use definition to stick to spec terms?

Reply:
Fixed. Decided to use term "field definition" instead of "field descriptor", incase it causes confusion with description (and couldn't think of a common word for both descirptor and difinition for "Don't leave empty lines between the description and the corresponding definition"). Is that okay?

Reply:
LGTM.